### ISSUE TITLE
Specifying ggplot options

### ISSUE THREAD

**Issue by pbreheny** [pbreheny/visreg#30]
Code like this should work, but doesn't:

```r
fit <- lm(Ozone ~ Solar.R + Wind*Temp, airquality)
v <- visreg(fit, "Wind", "Temp", gg=TRUE) + theme_bw()
```

This works:

```r
fit <- lm(Ozone ~ Solar.R + Wind*Temp, airquality)
v <- visreg(fit, "Wind", "Temp", plot=FALSE)
plot(v, gg=TRUE) + theme_bw()
```

But the first one would be cleaner / easier / more user-friendly.

**Comment by agilebean** [comment]
I can confirm that the option "gg=TRUE" does not work - unfortunately.
`plot(v, gg=TRUE) `
works, but is not a ggplot object.
So any other ggplot option (like `+ theme_bw()`) is disregarded

It would be great if this could be fixed!

**Comment by pbreheny** [comment]
This is fixed in visreg 2.4-0, making its way to CRAN now.
